### PR TITLE
AP_Temperature: slow down temp driver thread

### PR DESCRIPTION
**Pull request by magicrub**
slow down temp driver threads and some minor cleanup

### CODE REVIEW

File: libraries/AP_TemperatureSensor/AP_TemperatureSensor_MAX31865.cpp [hunk 1]
```diff
@@ -94,7 +94,7 @@ void AP_TemperatureSensor_MAX31865::init()
     _dev->set_speed(AP_HAL::Device::SPEED_HIGH);
 
     /* Request 10Hz update */
```
Comment (IamPete1) [inline libraries/AP_TemperatureSensor/AP_TemperatureSensor_MAX31865.cpp:96]: Comment needs a update, you might as well go to `200` too, less math is more better.

Comment (magicrub) [reply libraries/AP_TemperatureSensor/AP_TemperatureSensor_MAX31865.cpp]: fixed

**Comment by magicrub** [comment]
@IamPete1 all good to merge?

**Comment by magicrub** [comment]
To my knowledge, I'm the only one using there new drivers and the only one to test them so I'll go ahead and merge.

**Pull request merged**